blocks/image-compare/assets/juxtapose.css

```diff
@@ -0,0 +1,285 @@
+/**
+ * Juxtapose - v1.2.0 - 2017-12-18
+ * Copyright (c) 2017 Alex Duner and Northwestern University Knight Lab
+ *
+ * This code is governed by the terms of the Mozilla Public License (MPL) v2.0,
+ * which is available here: https://github.com/NUKnightLab/juxtapose/blob/master/LICENSE
+ *
+ * This code is incorporated into Image Compare plugin, which is licensed under
+ * GPLv2+, however you may use Juxtapose code separately under the terms of its
+ * original MPL 2.0 license if you wish.
+ */
+
+div.juxtapose {
+	width: 100%;
+	font-family: Helvetica, Arial, sans-serif;
+}
+
+div.jx-slider {
+	width: 100%;
+	height: 100%;
+	position: relative;
+	overflow: hidden;
+	cursor: pointer;
+	color: #f3f3f3;
+}
+
+div.jx-handle {
+	position: absolute;
+	height: 100%;
+	width: 40px;
+	cursor: col-resize;
+	z-index: 15;
+	margin-left: -20px;
+}
+
+.vertical div.jx-handle {
+	height: 40px;
+	width: 100%;
+	cursor: row-resize;
+	margin-top: -20px;
+	margin-left: 0;
+}
+
+div.jx-control {
+	height: 100%;
+	margin-right: auto;
+	margin-left: auto;
+	width: 3px;
+	background-color: currentColor;
+}
+
+.vertical div.jx-control {
```

Comment:
I imagine this `.vertical` class comes from the Juxtapose library and so we probably can't/don't want to customize it. But ideally this would be prefixed.